mis_builder/readme/newsfragments/590.bugfix

```diff
@@ -0,0 +1 @@
+Restore compatibility with python 3.9
```

Comment:
may be move the file to 597.bugfix to point to the proper PR? Other than that LGTM.

Reply:
good catch